components/permission-provider/org.wso2.carbon.analytics.permissions/src/main/java/org/wso2/carbon/analytics/permissions/PermissionProvider.java

```diff
@@ -37,6 +37,13 @@
      */
     void addPermission(Permission permission) throws PermissionException;
 
+    /**
+     * Check permission.
+     *
+     * @param permission
```

Comment:
Can you please explain what the parameter is (i.e. in Javadoc comments)

Reply:
Fixed 4911621